### PR TITLE
Create whattyu.json

### PR DESCRIPTION
**Pull request by atatolgadln**

### CODE REVIEW

**Review by hackermondev — CHANGES_REQUESTED** [review]
You need records?

**Comment by whattyu** [comment]
@hackermondev can u look

**Comment by whattyu** [comment]
@hackermondev @TheBotlyNoob please look

**Comment by mtgsquad** [comment]
@whattyu If you mention another maintainer, you will be barred from the service for 1 full month.

**Comment by MaskDuck** [comment]
well, @whattyu remember we have real life ụnlike you.

we're not paid to run this service, it's entirely volunteer based. 

if we are too busy to review, you must respect that. we'll review yours when we have times.

File: domains/whattyu.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "My Website",
+  "repo": "https://github.com/whattyu/website",
+  "owner": {
+    "username": "whattyu",
+    "email": "dulunatatolga@gmail.com"
+  },
+  "record": {
+    "CNAME": "whattyu.vercel.app"
```
Comment (MaskDuck) [inline domains/whattyu.json:9]: vercel cant be used sowwy

instead try [railway](https://railway.app)

**Review by MaskDuck — APPROVED** [review]

**Review by MaskDuck — CHANGES_REQUESTED** [review]
bruh i meant request changes

File: domains/whattyu.json (hunk 1 above)
Comment (whattyu) [reply domains/whattyu.json:9]: railway gives me an error but in vercel isn't

Comment (whattyu) [reply domains/whattyu.json:9]: github pages too

Comment (MaskDuck) [reply domains/whattyu.json:9]: idc

Comment (MaskDuck) [reply domains/whattyu.json:9]: our service doesn't work with vercel/netlify, sowwy :(

**Pull request closed without merging**